sdk/python/lib/pulumi/automation/_local_workspace.py

```diff
@@ -414,8 +414,11 @@ def select_stack(self, stack_name: str) -> None:
     def remove_stack(self, stack_name: str) -> None:
         self._run_pulumi_cmd_sync(["stack", "rm", "--yes", stack_name])
 
-    def list_stacks(self) -> List[StackSummary]:
-        result = self._run_pulumi_cmd_sync(["stack", "ls", "--json"])
+    def list_stacks(self, include_all: Optional[bool] = None) -> List[StackSummary]:
```

Comment:
Curious if there's a reasoning for calling this `include_all`, while we just call the argument `all` in the other languages?

Reply:
I believe `all` is considered somewhat of a keyword in python. When I attempted to use `all` before, it caused the linting to fail.